x-pack/test/alerting_api_integration/observability/custom_threshold_rule/documents_count_fired.ts

```diff
@@ -95,14 +98,14 @@ export default function ({ getService }: FtrProviderContext) {
                 threshold: [1, 2],
                 timeSize: 1,
                 timeUnit: 'm',
-                metrics: [{ name: 'A', filter: '', aggType: Aggregators.COUNT }],
+                metrics: [{ name: 'A', filter: 'container.id:*', aggType: Aggregators.COUNT }],
               },
             ],
             alertOnNoData: true,
             alertOnGroupDisappear: true,
             searchConfiguration: {
               query: {
-                query: '',
+                query: 'host.name:*',
```

Comment:
Why does `query` now have this value? Where did it come from?

Reply:
I've added it to test passing the query to the viewInApp URL.